expat/configure.ac

```diff
@@ -119,16 +119,21 @@ AC_C_CONST
 AC_TYPE_SIZE_T
 
 AC_ARG_WITH([xmlwf],
-  [AS_HELP_STRING([--without-xmlwf], [do not build xmlwf])], [],
+  [AS_HELP_STRING([--without-xmlwf], [do not build xmlwf])],
+  [],
   [with_xmlwf=yes])
 AM_CONDITIONAL([WITH_XMLWF], [test x${with_xmlwf} = xyes])
 
 AC_ARG_WITH([examples], [
-AS_HELP_STRING([--without-examples], [do not build examples @<:@default=included@:>@])], [], [with_examples=yes])
+AS_HELP_STRING([--without-examples], [do not build examples @<:@default=included@:>@])],
+  [],
+  [with_examples=yes])
 AM_CONDITIONAL([WITH_EXAMPLES], [test x${with_examples} = xyes])
 
 AC_ARG_WITH([tests], [
-AS_HELP_STRING([--without-tests], [do not build tests @<:@default=included@:>@])], [], [with_tests=yes])
+AS_HELP_STRING([--without-tests], [do not build tests @<:@default=included@:>@])],
```

Comment:
`AS_HELP_STRING` is a parameter to `AC_ARG_WITH` but indendation does not match any of rule 2 at #390.